### PR TITLE
Fix service, improve CI

### PR DESCRIPTION
**Pull request by Jasstkn**
#### What this PR does / why we need it:
- fix service for frontend: we can should define ClusterIP as a default, during chart installation user can change service type
- add CODEOWNERS file to automatically assign PRs
- add pre-commit to enforce docs update and helm-lint before commit
- update staled READMEs

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/litmus-2-0-0-beta/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# litmus-portal
+# litmus-2-0-0-beta
```
Comment (rajdas98) [inline charts/litmus-2-0-0-beta/README.md:1]: I think it should be Litmus-2.0.0-Beta1

Comment (rajdas98) [reply charts/litmus-2-0-0-beta/README.md:1]: same with other places.

**Review by rajdas98 — APPROVED** [review]
Other than, the naming change. PR looks good to me

File: charts/litmus-2-0-0-beta/README.md (hunk 1 above)
Comment (Jasstkn) [reply charts/litmus-2-0-0-beta/README.md:1]: It uses [chart name](https://github.com/litmuschaos/litmus-helm/blob/master/charts/litmus-2-0-0-beta/Chart.yaml#L4)

Version is placed [correctly](https://github.com/litmuschaos/litmus-helm/blob/f2736d30dc16c7731cfe8f1cb9037d47096c218a/charts/litmus-2-0-0-beta/README.md)

Comment (rajdas98) [reply charts/litmus-2-0-0-beta/README.md:1]: version is- `2.0.0-Beta1` not `2.0.0-beta1`

Comment (Jasstkn) [reply charts/litmus-2-0-0-beta/README.md:1]: Hm, I didn't change the letter: https://github.com/litmuschaos/litmus-helm/blob/master/charts/litmus-2-0-0-beta/Chart.yaml#L5, but ok, I can fix that.

Comment (Jasstkn) [reply charts/litmus-2-0-0-beta/README.md:1]: @rajdas98 fixed. re-check, please

**Comment by rajdas98** [comment]
Thanks @Jasstkn. @ispeakc0de Can you review this PR and merge it to the master?

**Review by ksatchit — APPROVED** [review]

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**